File: C/tempo_de_jogo.c
#include <stdio.h>

int main()
{
    int hi, hf, duracao;

    printf("Hora inicial: ");
    scanf("%d", &hi);


    printf("Hora final: ");
    scanf("%d", &hf);

    if (hi < hf){
        duracao = hf - hi;
    } else {
        duracao = 24 - (hf - hi);
    }

    printf("O JOGO DUROU %d HORA%(S%)", duracao);

    return 0;
}

File: Python/tempo_de_jogo.py
hi = int(input("Hora inicial: "))
hf = int(input("Hora final: "))

if hi < hf:
    duracao = hf - hi
else:
    duracao = 24 - (hi - hf)

print(f"O JOGO DUROU {duracao} HORA(S)")